ARC specification.md

```diff
@@ -207,25 +208,6 @@ Notes:
 
 - It is expected that workflow and tool descriptions are authored semi-automatically, e.g. using the [arcCommander](https://github.com/nfdi4plants/arcCommander) tool.
```

Comment:
Is the reference to the arc commander good here, afaik we do not support this currently?